tests/acceptance/features/webUIFavorites/favoritesFile.feature

```diff
@@ -14,33 +14,28 @@ Feature: Mark file as favorite
   Scenario: mark files as favorites
     When the user marks file "data.tar.gz" as favorite using the webUI
     And the user marks file "data.zip" as favorite using the webUI
-    Then file "data.tar.gz" should be marked as favorite on the webUI
-    And file "data.zip" should be marked as favorite on the webUI
-    When the user reloads the current page of the webUI
-    Then file "data.tar.gz" should be marked as favorite on the webUI
-    And file "data.zip" should be marked as favorite on the webUI
+    Then file "data.tar.gz" should be marked as favorite
```

Comment:
Checking via propfind not to do gymnastics with either opening sidebar or opening file actions dropdown.

Reply:
you are now making sure that the backend has saved the result as expected :+1: 
but you don't know if the UI has updated correctly :disappointed: 
we had bugs that would not have caught by that kind of tests, that why I would suggest to check as many results of your action as you can. In this case at least the result in the backend and the immediate changes in the UI

Reply:
Ok, I'll use the sidebar for checking then. Do you think it would be a good idea to unite the check for UI and propfind? Or rather keep it separated?

Reply:
I would keep the separate, to make clear what is checked and there might be situations where you only want to check one thing, e.g. fav. is set by the API (other client) and you want to make sure the UI shows the correct result